Sources/XMTPRust/SwiftBridgeCore.swift

```diff
@@ -1,4 +1,6 @@
+import Foundation
```

Comment:
Looks like a dupe import here

Reply:
Good eyes, I need to go revisit my import injection code in libxmtp

Reply:
https://github.com/xmtp/libxmtp/pull/71